### PR TITLE
Add RSS feed hyperlink

### PR DESCRIPTION
**Pull request by huey735**
I added hyperlinks to /blog/index.html and /_layouts/post.html.
I didn't make any change to /_layouts/post-report.html.

### CODE REVIEW

**Comment by m52go** [comment]
@huey735 could we make the icon slightly smaller and better vertically aligned?

_Before_

Screenshot from 2019-08-01 13-18-55

_After_

Screenshot from 2019-08-01 13-18-40

May seem like a nit-pick for desktop but it makes a visible difference on smaller screens, especially phone screens. Below is the markup I used.

anchor tag -->

    display: inline-block;
    float: right;
    margin-top: 9px;

img tag -->

    width: 30px;

**Review by huey735 — COMMENTED** [review]
@m52go I added the css line inline, inside the html tags. I'm not sure if that's what you meant.

File: _layouts/post.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
   <article id="post-802" class="post-802 post type-post status-publish format-standard hentry category-news">
     <div class="dets_wrap">
       <div class="dets">
+        <a href="{{ site.url }}/blog/feed.atom" style="display: inline-block; float: right; margin-top: 4px;"><img src="{{ site.url}}/images/logo-rss.svg" class="rounded float-right" style="width: 30px;"></a>
```
Comment (m52go) [inline _layouts/post.html]:
```suggestion
       <a href="{{ site.url }}/blog/feed.atom" class="rss-link post"><img src="{{ site.url}}/images/logo-rss.svg"></a>
```

File: blog/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: Blog &lsaquo; Bisq - The decentralized Bitcoin exchange
 ---
-
+<a href="{{ site.url }}/blog/feed.atom" style="display: inline-block; float: right; margin-top: 9px;"><img src="{{ site.url}}/images/logo-rss.svg" class="rounded float-right" style="width: 30px;"></a>
```
Comment (m52go) [inline blog/index.html]:
```suggestion
<a href="{{ site.url }}/blog/feed.atom" class="rss-link blog-index"><img src="{{ site.url}}/images/logo-rss.svg"></a>
```

**Review by m52go — CHANGES_REQUESTED** [review]
@huey735 sorry I should've been more specific. I'd prefer to see CSS markup kept separate from HTML markup. I suggested HTML changes in-line. Here is the accompanying CSS markup:

```
.rss-link {
    float: right;
    width: 30px;
    height: auto;
}

.rss-link img {
    width: 100%;
}

.rss-link.blog-index {
    margin-top: 9px;
}

.rss-link.post {
    margin-top: 4px;
}

.post.type-post .dets h2, h1#blog-index-header {
    padding-right: 50px;
}
```

If you accept my HTML suggestions and add this markup to `css/styles.css` (ideally in the section marked 'blog' after line 707), it should be good.

**Comment by m52go** [comment]
ACK

**Comment by m52go** [comment]
I forgot to change the ID of the blog index page header in my suggestions, so I'll follow this merge with that small tweak.

**Pull request merged**